src/mistral_common/protocol/instruct/normalize.py

```diff
@@ -370,6 +398,36 @@ def normalizer() -> "InstructRequestNormalizerV13":
             InstructRequest[UATS, Tool],
         )
 
+    def _aggregate_role(
+        self, messages: List[UATS], role: Optional[Roles], latest_call_ids: List[str]
+    ) -> Sequence[UATS]:
+        if role == Roles.system:
+            return [
+                self._system_message_class(content=self._aggregate_content_chunks(message.content))
+                for message in messages
+                if isinstance(message, self._system_message_class)
+            ]
+        return super()._aggregate_role(messages, role, latest_call_ids)
+
+    def _aggregate_content_chunks(  # type: ignore[override]
```

Comment:
```suggestion
    def _aggregate_content_chunks(
```

no type ignore please